docs/stdlib/array.rst

```diff
@@ -77,9 +77,8 @@ For example:
 Empty arrays
 ^^^^^^^^^^^^
 
-An empty array can also be created, but it must be used together with
-a type cast, since EdgeDB cannot infer the type of an array that contains no
-elements.
+An empty array can also be created, but it must be used with
```

Comment:
```suggestion
You can also create an empty array, but it must be done by providing the type
information using type casting. EdgeDB cannot infer the type of an empty array
created otherwise. Example:
```